### ISSUE TITLE
Is Nuke still working on the mod?

### ISSUE THREAD

**Issue by xgiggsbr** [Nukem9/dlssg-to-fsr3#416]
Hey guys, can you tell me if nukem is still working on the mod, this mod is the best of all, I hope it's still being worked on

**Comment by Weather-OS** [comment]
No. This was project was allegedly sabotaged by NVIDIA themselves. 
LukeFZ is still updating his FSR2FSR3 mods if you're interested.

**Comment by xgiggsbr** [comment]
I understand, I've already tested the lukefz one, it's good but I prefer nukem. Thank you for the answer

**Comment by TimmyP777** [comment]
FSR3.1 decouples from upscaler. This essentially means that no developer in their right mind isn't going DLSS3FG over FSR3.1FG unless sponsorship, and even then we have mods.

I personally believe that, given that there are like five total places where this hobby is discussed now, that Nvidia has plants to misinform, "de-educate," and do everything possible to lead a potential 4000 series buyer to a purchase because this mod exposes more than just what you see. AMD does it too, playing up "vram" this without educating anyone on anything about how a buffer works.

The "updated OFA" is a load of horse-sh*t, and it was on 3000 series (imo). Nvidia is doing everything they can to hide the fact that upgrading to 4000 series for frame generation has ALWAYS been pointless (sans 4090). This was already all being done in vr.

Have a "desire" to move from a 3070 to a 4070 for that sweet FG? I did too. This mod legit removed it completely.

**Comment by xgiggsbr** [comment]
This mod showed that it is possible for owners of the 20 and 30 series to have an FG, Nvidia could create it via software for these series, however the 40 series is being a failure and they still want to sell it. I am very sad to know that the mod It will be discontinued and FSR 3.1 will take a long time to reach games

**Comment by TimmyP777** [comment]
It was already shown possible with VR. 4000 series was marketed on a "bogus" feature, and I imagine Nvidia could find themselves  in trouble somehow if this became common knowledge, but "knowledge" is being erased on the internet now (thanks reddit\facebook! You ruined the internet!), and this mod is a victim of it.

FSR3.1 is right around the corner. Horizon FW dev said they are holding off until it drops, then they will implement it.

**Comment by xgiggsbr** [comment]
It's getting ugly for Nvidia now that AMD releases its technologies to everyone, without exception, Nvidia only aims for money

**Comment by Aidorin0** [comment]
Isn't the mod already finished? It just works in almost all games with native dlss FG, so no need to 'still working' on it.

**Comment by TimmyP777** [comment]
For the most part. Generally a dev wont leave a complete project at version .9 tho.

**Comment by Erebus1245** [comment]
I asked Nukem on Discord a while back and he said that there is not much else to implement for now and the stuff he could do  (Vulkan, streamline related bugs) is too hard/annoying to fix for him, that doesn't mean the mod is abandoned though he's just working on other stuff like his HFW mod atm. I'm sure he will come back to it when AMD releases FSR 3.1.

**Comment by TimmyP777** [comment]
Yeah cuz he'd tell you Nvidia is on his ass for sure

**Comment by TimmyP777** [comment]
There is nothing to follow dude. Ghost of Tsushima is out with FSR3.1. This means decoupled upscaler\frame generator.

Nvidia isn't going to risk forcing DLSS3-FG exclusivity anymore; the "publicity" wont be worth it.

3000 to 4000 was a generally waste for anyone that did. Confirmed. Glad everyone else, who remained ignorant to this mod, gets to see it now.

**Comment by Sousaboi** [comment]
Nah I just couldn’t find the “get updates from this issue/thread” so I just said that lmao. Thx for the info tho
